net.hovancik.Stretchly.yaml

```diff
@@ -17,6 +17,9 @@ finish-args:
   # - --socket=fallback-x11
   # - --socket=wayland
   - --socket=pulseaudio
+  # Access DBus for idle status
+  - --system-talk-name=org.freedesktop.Notifications
```

Comment:
It seems weird that this one would require access to the system session bus, since I can't seem to find any flatpaks that do that for `org.freedesktop.Notifications`.

Reply:
Agreed, I fully expected it to use the session bus. The errors only stop however when granting it the system bus.

Additional weirdness is that yesterday when I created the PR and tested, DnD seemed to be working correctly; today it doesn't seem to block the reminders even though it doesn't throw errors on startup. :woman_shrugging: 

May need further debugging or logging statements to figure out why it is showing even though DnD is active.